docs/preview/features/powershell/azure-storage.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Scripts related to interacting with Azure Storage"
+title: "Scripts related to interacting with Azure Table Storage"
```

Comment:
This file will be the "overview"-page for all storage-related modules. (related to #67).
I would use this page to provide an overview of the listed modules and explain how to install the meta package `Arcus.Scripting.Storage` (once it exists)

```suggestion
title: "Scripts related to interacting with Azure Storage"
```

The section below would look like this:
 # Azure Table Storage
  This module provides the following capabilities:
  ... -> link to the table storage-documentation page to the correct section.

 # Azure File Share
  This module provides the following capabilities:
  ... -> link to the table storage-documentation page to the correct section.

Reply:
I updated this page with a menu like you suggested; also with the blob storage I got from `master`.